modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -43,10 +43,10 @@ $ oc patch Network.config.openshift.io cluster \
   --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
 ----
 
-. To confirm the migration disabled the OpenShift SDN default CNI network provider, enter the following command. It might take several moments for all the OpenShift SDN Pods to stop.
+. To confirm the migration disabled the OpenShift SDN default CNI network provider and removed all OpenShift SDN Pods, enter the following command. It might take several moments for all the OpenShift SDN Pods to stop.
 +
 ----
-$ oc get pod -n openshift-sdn --watch
+$ watch oc get pod -n openshift-sdn
```

Comment:
@dlbewley, what makes this preferable than `--watch`?

Reply:
The condition the user wants to see is that all the pods are gone. Using `--watch` simply emits a stream of changes . It is much more obvious when the pods are gone  with `watch <cmd>`, particularly if there are many pods (nodes). 

If watch can be assumed to be broadly available I think the output is much more clear.